hack/undeploy-ocp.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o nounset
+set -o pipefail
+
+excluded_files=("01_namespace.yaml")
+expected_error="delete leaderworkersetoperators.operator.openshift.io cluster"
+
+for file in deploy/*; do
```

Comment:
I don't think we need this iteration. We know that simply deleting openshift-lws-operator via `oc delete ns openshift-lws-operator` will prune most of the part. We just additionally need to remove the cluster scoped ones and that's all

Reply:
When can delete crds  $ oc get crds | grep leader
leaderworkersetoperators.operator.openshift.io                    2025-07-22T02:25:00Z
leaderworkersets.leaderworkerset.x-k8s.io                         2025-07-22T02:25:20Z

Reply:
I think, deleting them one by one gives us the control instead of an iteration.